proposals/0448-type-variable-scoping.rst

```diff
@@ -104,12 +105,19 @@ synonym of ``-XScopedTypeVariables``. Under this change, that would no longer be
 
 This component of this proposal is taken
 from the not-yet-accepted proposal `#238`_, changing the name of what I now call
-``-XExtendedForAllScope``, and simplifying the binding story around pattern signatures
-(getting rid of ``-XPatternSignatureBinds``). This part of the proposal also introduces
-a new warning ``-Wpattern-signature-binds`` (available only in ``-Weverything``) as a
-new way of handling the pattern-signature-binding part of `#285`_.
+``-XExtendedForAllScope``.
 
-This component includes the ``-XNoImplicitForAll`` of `#285`_ unchanged.
+``-XImplicitBinds`` is the combination of ``-XImplicitBindsForAll`` and ``-XPatternSignatureBinds``) from accepted
+proposal `#285`_, and differ from that proposal in that:
+
+  - The extensions are combined for fewer knobs when the motivations are the same.
+
+  - ``RULES`` was forgotten as an example.
+
+  - Examples of implicit binds falsely categorized as pattern signature binds are now explicitly included.
```

Comment:
Concretely, this means that in `data A (a :: k)`, `k` is now in the same category as kind signature occurrences?

Reply:
```suggestion
  - Examples of implicit binds falsely categorized as pattern signature binds are now properly included under the proposed change specification.
```